charts/netbox-operator/values.yaml

```diff
@@ -0,0 +1,378 @@
+# Default values for NetBox Operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+## @section Global parameters
+## Global container image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global container image parameters: imageRegistry, imagePullSecrets and storageClass
+
+## @param global.imageRegistry Global container image registry
+## @param global.imagePullSecrets Global container registry secret names as an array
+## @param global.storageClass Global StorageClass for Persistent Volume(s)
+##
+global:
+  imageRegistry: ""
+  ## E.g.
+  ## imagePullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  imagePullSecrets: []
+  storageClass: ""
+
+## @section Common parameters
+
+## @param nameOverride String to partially override common.names.fullname
+##
+nameOverride: ""
+## @param fullnameOverride String to fully override common.names.fullname
+##
+fullnameOverride: ""
+## @param commonLabels Labels to add to all deployed objects
+##
+commonLabels: {}
+## @param commonAnnotations Annotations to add to all deployed objects
+##
+commonAnnotations: {}
+## @param clusterDomain Kubernetes cluster domain name
+##
+clusterDomain: cluster.local
+## @param extraDeploy Array of extra objects to deploy with the release
+## Example:
+## extraDeploy:
+##   - |
+##     apiVersion: v1
+##     kind: ConfigMap
+##     metadata:
+##       name: sso-pipeline-roles
+##       namespace: netbox
+##     data:
+##       sso_pipeline_roles.py: |
+##         from netbox.authentication import Group
+##         ...
+##
+extraDeploy: []
+
+## @section NetBox Operator Image parameters
+## @param image.registry Image registry
+## @param image.repository Image repository
+## @param image.tag Image tag
+## @param image.digest Image digest in the way sha256:aa...
+## @param image.pullPolicy MariaDB image pull policy
+## @param image.pullSecrets Specify docker-registry secret names as an array
+##
+image:
+  registry: ghcr.io
+  repository: netbox-community/netbox-operator
+  pullPolicy: IfNotPresent
+  ## Defaults to '{{ .Chart.AppVersion }}'
+  ##
+  tag: ""
+  ## If set, override the tag
+  ##
+  digest: ""
+  ## Optionally specify an array of imagePullSecrets (secrets must be manually created in the namespace)
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## Example:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  pullSecrets: []
+
+## @section NetBox Operator Configuration parameters
+
+## NetBox Operator settings based on environment variables
+caCert: ""
+host: ""
+https: false
+restorationHashFieldName: "netboxOperatorRestorationHash"
```

Comment:
We already default this in the Operator. Maybe we can comment this out in the values.yaml?

Reply:
```suggestion
restorationHashFieldName: ""
```